client/components/tinymce/index.jsx

```diff
@@ -286,7 +302,11 @@ module.exports = React.createClass( {
 			body_class: 'content post-type-post post-status-draft post-format-standard locale-en-us',
 			add_unload_trigger: false,
 
-			setup: setup
+			setup: setup,
+
+			templates: [
```

Comment:
The template plugin is loaded in another iframe which we don't have direct access to. Not sure how we can get the template preview to work according to the custom page type (ie. Post Category).

![screen shot 2016-02-24 at 4 57 58 pm](https://cloud.githubusercontent.com/assets/2896608/13306363/cbe69e98-db17-11e5-9b6f-19fadcc55c29.png)